static-parts/index-start.html

```diff
@@ -31,6 +34,7 @@ <h1 class="logo" style="margin-top: -0.8rem; text-align:center;"><a style="text-
     </header>
     <div class="row" style="height: calc(100vh - 5.875rem);overflow: hidden;">
       <input id="doc-drawer-checkbox" class="drawer" value="on" type="checkbox">
-        <nav class="col-md-4 col-lg-3" style="border-top: 0">
+        <nav id="snippetListDrawer" class="col-md-4 col-lg-3" style="border-top: 0">
```

Comment:
you should no longer need to touch this line since you no longer need an id attribute

Reply:
Oh yeah, that's actually right. Thanks for noticing @skatcat31